cmd/bigslice/ec2.go

```diff
@@ -74,6 +75,10 @@ func setupEc2Cmd(args []string) {
 		must.True(os.IsNotExist(err), err)
 	}
 
+	if region, ok := profile.Get("aws/env.region"); ok && len(region) > 0 {
+		must.Nil(profile.Set("bigmachine/ec2system.default-region", strings.Trim(region, `"`)))
```

Comment:
Why do we need to `Trim` here?

(Sorry I missed this the first time around).

Reply:
Json double-escaping... the region can end up being "\"us-west-2\"".

Reply:
Won't that only happen if we muck up escaping somewhere else?  It seems that we [do something similar with`ec2system.security-group`](https://github.com/grailbio/bigslice/blob/0d07a334180d1918b8bbf594e14628ceacdd1745/cmd/bigslice/ec2.go#L82), though I'm not sure why there either.

Reply:
I don't really know all of the code paths that lead to a value being setup here, nor how quotes are expected to be handled by grailbio/config but it's certainly the case the escaped quotes end up here. So in the interests of expediency, and knowing that the region will not have internal quotes, it seemed reasonable to just remove them here.